chain/db/slashing.go

```diff
@@ -0,0 +1,11 @@
+package db
```

Comment:
Let's handle the case when a validator got jailed too. Jail should become true in that case?